app/models/sitemap.rb

```diff
@@ -0,0 +1,2 @@
+class Sitemap < ApplicationRecord
+end
```

Comment:
Is there a reason for this file?

Reply:
Its just the model that the sitemap xml data gets stored in. The SitemapBuilder class is more like a service class that houses all of the methods required to put some data into this.